src/minihits.cpp

```diff
@@ -12,44 +12,9 @@
 
 #define assert_msg(x) !(std::cerr << "Assertion failed: " << x << std::endl)
 
-MinimizerHits::MinimizerHits(const uint32_t &num_hits) {
-    uhits.reserve(num_hits);
-}
-
-void MinimizerHits::clear() {
-    /*for (const auto &c: hits)
-    {
-        delete c;
-    }*/
-    hits.clear();
-
-    /*for (const auto &c: uhits)
-    {   
-        delete c;
-    }*/
-    uhits.clear();
-}
-
-MinimizerHits::~MinimizerHits() {
-    clear();
-}
-
-void MinimizerHits::add_hit(const uint32_t i, const Minimizer &m, const MiniRecord *r) {
-    MinimizerHitPtr mh(std::make_shared<MinimizerHit>(i, m, r));
-    uhits.insert(mh);
-}
-
-void MinimizerHits::sort() {
-    if (hits.max_size() > uhits.size()) {
-        hits.insert(uhits.begin(), uhits.end());
-        assert(hits.size() == uhits.size() ||
-               assert_msg("Expected uhits.size()=" << uhits.size() << " to equal hits.size()=" << hits.size()));
-        uhits.clear();
-    } else {
-        std::cerr << "Could not create a set big enough for " << uhits.size() << " elements. The max size is "
-                  << hits.max_size() << std::endl;
-        std::exit(EXIT_FAILURE);
-    }
+void MinimizerHits::add_hit(const uint32_t i, const Minimizer &minimizerFromRead, const MiniRecord &minimizerFromPRG) {
```

Comment:
Please change variable names to snake_case

Reply:
Done in https://github.com/leoisl/pandora/commit/5b329a78d89eaa0f85592258a8fb378677f71587